<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kscreensystem</class>
 <widget class="QWidget" name="kscreensystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="systemConfigCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable screen configuration for system</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="systemConfigLayout">
    <item>
     <widget class="QRadioButton" name="newusersRadioButton">
      <property name="text">
       <string>Set to new users the current configuration by default</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="allusersRadioButton">
      <property name="text">
       <string>Force current configuration to all users</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
